players.sql

```diff
@@ -0,0 +1,67 @@
+CREATE TABLE players (
+  id integer UNIQUE NOT NULL,
+  last_name text NOT NULL,
+  first_name text NOT NULL,
+  position varchar(3),
+  jersey_number integer,
+  weight_lb integer NOT NULL,
+  height_in integer NOT NULL,
+  date_of_birth date NOT NULL
+);
+
+INSERT INTO players VALUES (
+  1,
+  'Fisher', 'Derek', 'QB',
+  2, 210, 74, '1999-01-02'
+);
+
+INSERT INTO Players VALUES (
+  2,
+  'Cunningham', 'Reginald', 'DE',
+  55, 240, 75, '1998-04-03'
+);
+
+INSERT INTO players VALUES (
+  3,
+  'Jackson', 'Stephen', 'WR',
+  13, 184, 76, '1998-10-13'
+);
+
+INSERT INTO players VALUES (
+  4,
+  'Colton', 'Ryan', 'K',
+  2, 210, 74, '1999-01-02'
+);
+
+INSERT INTO players VALUES (
+  5,
+  'Wright', 'Alexander', 'RB',
+  23, 190, 70, '1999-03-13'
+);
+
+INSERT INTO Players VALUES (
+  6,
+  'Smalls', 'Elliot', 'OBL',
+  33, 215, 69, '1999-02-23'
+);
+
+ALTER TABLE players ADD CONSTRAINT position NOT NULL;
+ALTER TABLE players ADD CONSTRAINT jersey_number NOT NULL;
```

Comment:
Setting a `NOT NULL` constraint has a [slightly different syntax](https://www.postgresql.org/docs/12/ddl-alter.html#DDL-ALTER-ADDING-A-CONSTRAINT)
```suggestion
ALTER TABLE players ALTER COLUMN position SET NOT NULL;
ALTER TABLE players ALTER COLUMN jersey_number SET NOT NULL;
```